### PR TITLE
Server error handling 

### PR DESCRIPTION
**Pull request by ajaypratap003**
### Type of change

- Enhancement / new feature
- Refactoring

### Description
1. Created custom hook for mutation queries
2. Created components for network error and server errors
3. Implemented bulk mutation queries for delete address/address space
4. Created context state reducer module to handle the states globally for error handling

### Checklist

- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by ajaypratap003** [comment]
@enmasse-ci run tests profile=shared testcase=web.**

**Comment by ajaypratap003** [comment]
@enmasse-ci run tests profile=shared testcase=web.**

**Comment by ajaypratap003** [comment]
@enmasse-ci run tests profile=shared testcase=web.**

**Comment by ajaypratap003** [comment]
@enmasse-ci run tests profile=shared testcase=web.**

File: console/console-init/ui/src/pages/AddressSpaceDetail/AddressList/AddressListPage.tsx [hunk 1]
```diff
@@ -117,9 +146,6 @@ export const AddressListPage: React.FunctionComponent<IAddressListPageProps> = (
     setOnCreationRefetch(false);
   }
 
-  if (loading) return <Loading />;
```
Comment (drcoolsanjeev) [inline console/console-init/ui/src/pages/AddressSpaceDetail/AddressList/AddressListPage.tsx]: revert this loading, as it handles server response

**Comment by ajaypratap003** [comment]
@enmasse-ci run tests profile=shared testcase=web.**

**Comment by rkpattnaik780** [comment]
Not able to purge an address from the AddressList using the Kebab.

Screenshot from 2020-02-27 17-46-42

**Comment by ajaypratap003** [comment]
> Not able to purge an address from the AddressList using the Kebab.
> 
> Screenshot from 2020-02-27 17-46-42

This issue has fixed and changes already commit.

**Comment by ajaypratap003** [comment]
@enmasse-ci run tests profile=shared testcase=web.**

**Pull request merged**